pkgs/by-name/go/google-photos-takeout-helper/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildDartApplication,
+  fetchFromGitHub,
+}:
+
+buildDartApplication rec {
+  pname = "google-photos-takeout-helper";
```

```suggestion
  pname = "gpth";
```
and change the attibute name.
This is the name other distros use: https://repology.org/project/gpth